src/main/kotlin/nl/info/client/brp/BrpClientService.kt

```diff
@@ -54,22 +62,27 @@ class BrpClientService @Inject constructor(
 
     fun queryPersonen(personenQuery: PersonenQuery): PersonenQueryResponse =
         updateQuery(personenQuery).let {
-            personenApi.personen(it)
+            personenApi.personen(
+                personenQuery = it,
+                purpose = purposeSearch.validatePurpose("brp.doelbinding.zoekmet")
+            )
         }
 
     /**
-     * Retrieves a person by burgerservicenummer from the BRP Personen API.
+     * Retrieves a person by burgerservicenummer from the BRP Personen API by validating
+     * the appropriate doelbinding purpose configuration.
```

Comment:
Very hard to interpret `doelbinding purpose`. I would leave this in Dutch entirely. If you want to translate it to English, fine, but then also translate `doelbinding`? But I would leave it in Dutch really because in this case I think translating it would only add confusion.

Also the text is not correct I would say. We are not retrieving a person _by_ validating it. We are retrieving a person from the BRP API and while doing so we are adding a 'doelbinding'. Right?